### DISCUSSION TITLE
Street texture is 100% similar to side walk texture, I need computer use some clue to segment better

### DISCUSSION THREAD

**Discussion by damavand1** [ultralytics/ultralytics#15391] [General]
Hi ultralytics team

please attention to this pictures and guess what your YoloV8 segment?
I need smart segmentation, that segment using clues 

![20240721_190147](https://github.com/user-attachments/assets/82825440-eb32-483f-900a-75e4910fc086)

![20240721_190145](https://github.com/user-attachments/assets/2b26e3ea-08d3-4276-84aa-56096fdb8731)

![20240721_190134](https://github.com/user-attachments/assets/f8906f16-e98a-421e-99b1-5b8083187aae)

![20240721_185952](https://github.com/user-attachments/assets/8b85a056-aebf-4474-ba6c-6fac64deb174)

**Comment by glenn-jocher** [comment]
Thank you for sharing the images. YOLOv8's segmentation performance can be influenced by the dataset and training process. To improve segmentation in scenarios with similar textures, consider augmenting your dataset with more diverse examples and using advanced data augmentation techniques. Additionally, fine-tuning the model with specific annotations for street and sidewalk textures can help. Ensure you're using the latest version of the Ultralytics package for the best performance. If the issue persists, please provide more details about your training setup.
